Documentation/issue-filing-guide.md

```diff
@@ -0,0 +1,41 @@
+Filing issues for .NET Core CLI
+===============================
+
+As you may notice based on our issues page, the CLI repo is what is known as a 
+"high-profile" and "high-volume" repo; we 
+get a lot of issues. This, in turn, may mean that some issues get 
+lost in the noise and/or are not reacted on with the needed speed. 
+
+In order to help with the above situation, we need to have a certain way to file 
+issues in order for the core team of maintainers can react as fast as (humanly)
```

Comment:
I think you need to fix this phrasing. ...in order for the core team of maintainers to react as fast as...